spec/spec_helper.rb

```diff
@@ -47,19 +47,19 @@
 
   # suppressing outputs during tests
 
-  config.before(:all) do
-    $origin_stdout = $stdout
-    $origin_stderr = $stderr
-    $stdout = StringIO.new
-    $stderr = StringIO.new
-  end
+  # config.before(:all) do
+  #   $origin_stdout = $stdout
+  #   $origin_stderr = $stderr
+  #   $stdout = StringIO.new
+  #   $stderr = StringIO.new
+  # end
 
-  config.after(:all) do
-    $stdout = $origin_stdout
-    $stderr = $origin_stderr
-    $origin_stdout = nil
-    $origin_stderr = nil
-  end
+  # config.after(:all) do
+  #   $stdout = $origin_stdout
+  #   $stderr = $origin_stderr
+  #   $origin_stdout = nil
+  #   $origin_stderr = nil
+  # end
```

Comment:
Feel free to remove any commented out code. Remember we can always get them back from Git history!

Reply:
Resolve at 4366f41b26aec5c9cbf1f087275c1ec47033cacb

Reply:
Resolved at 8c52707d7bd8ae4d77b90c08bc7555fcc3456aac